nilearn/_utils/data_gen.py

```diff
@@ -1241,9 +1272,34 @@ def _init_fields(subject, session, task, run):
     return fields
 
 
+def _write_bids_raw_anat(subses_dir: Path, subject: str, session: str) -> None:
```

Comment:
```suggestion
def _write_dummy_t1w(sessions_dir: Path, subject: str, session: str) -> None:
```
There's nothing BIDS specific in this dummy file.

Reply:
well the filename of the file generated follows a BIDS filename pattern.

sub-<sub-label>[_ses-<ses-label>]_T1w.nii.gz